src/frontend/Komponenter/Behandling/Inngangsvilkår/Aleneomsorg/ReadMoreMedAdresser.tsx

```diff
@@ -0,0 +1,72 @@
+import { ReadMore, Table } from '@navikt/ds-react';
+import React, { FC } from 'react';
+import styled from 'styled-components';
+import { useBehandling } from '../../../../App/context/BehandlingContext';
+import DataViewer from '../../../../Felles/DataViewer/DataViewer';
+import { useGjeldendeBarn } from '../../../../App/hooks/useGjeldeneBarn';
+import { IAdresse } from '../../../../App/typer/personopplysninger';
+
+const StyledReadMore = styled(ReadMore)`
+    --ac-read-more-text: var(--a-text-default);
+`;
+
+const ReadMoreMedAdresser: FC<{ barnIdent?: string }> = ({ barnIdent }) => {
+    const { personopplysningerResponse } = useBehandling();
+    const gjeldendeBarn = useGjeldendeBarn(barnIdent);
+
+    const sisteAdresse = (adresse: IAdresse[]) => {
+        return adresse[adresse.length - 1]?.visningsadresse;
+    };
+
+    return (
+        <DataViewer response={{ personopplysningerResponse }}>
+            {({ personopplysningerResponse }) => (
+                <StyledReadMore
+                    header="Ikke registrert på brukers adresse"
+                    size="small"
+                    defaultOpen={true}
```

Comment:
Ikke nødvendig å endre på dette, men man trenger bare å skrive defaultOpen slik:
```suggestion
                    defaultOpen
```